apps/element-storybook/src/accessibility.mdx

```diff
@@ -0,0 +1,46 @@
+import { Meta } from '@storybook/addon-docs';
+import storybook_accessibility from '../../../static/storybook_accessibility.png';
+
+<Meta title="Accessibility" />
+
+# Accessibility
+
+The `@availity/element` library is built with accessibility in mind.
+Specifically, we are following the 508 compliance guidelines. However,
```

Comment:
Can we add a link to the specific guidelines? I think we aim for WCAG 2.1 AA compliance, but I don't remember exactly 😅 
https://www.w3.org/TR/WCAG21/

Reply:
Confirmed with UX it is WCAG 2.1 AA and added to the docs.